calyx-frontend/src/ast.rs

```diff
@@ -341,6 +343,54 @@ pub enum Control {
         /// Attributes
         attributes: Attributes,
     },
+    /// Represents sequential composition of static control statements.
+    StaticSeq {
```

Comment:
One other way to do this is having an enum like this:
```rust
enum ControlType {
  Dynamic,
  Static { latency: u64 }
}
```

And then make all the enum variants for the `Control` enum specify a `ControlType`. This probably doesn't matter too much since the frontend is not often used by any other part of the compiler library.

Reply:
Possibly worth doing, but I will also note that while redundant, this makes it possible to write code that knows at a type level that it will only be working with static groups, rather than having to check the `ControlType` variant and error out in certain cases. Not sure if this is worth the repetition but I figure it's worth mentioning.

Reply:
Yeah, when going from AST -> IR, I actually use this idea to give an error when someone tries to put dynamic control in static context, so I'll prob keep as is. Not that it really matters either way, though.